docs/src/modules/redux/initRedux.js

```diff
@@ -37,18 +37,3 @@ function create(initialState) {
     compose(applyMiddleware(...middleware), devtools),
   );
 }
-
-export default function initRedux(initialState) {
-  // Make sure to create a new store for every server-side request so that data
-  // isn't shared between connections (which would be bad)
-  if (!process.browser) {
-    return create(initialState);
-  }
-
-  // Reuse store on the client-side
-  if (!global.__INIT_REDUX_STORE__) {
```

Comment:
Note for context.

The incentive for the global was to account for cases where Next.js doesn't bundle redux into the share assets. In the past, I had multiple cases where we would get a new instance of redux per page. I don't know if it's still a concern today.

Reply:
How would we get multiple instances? There's only a single store.

Reply:
Basically, Next.js/webpack would fail to identify Redux as module that needs to be put into the shared vendor. In some cases, I would have:

- shared_bundle.js
- page_a.js contains `docs/src/modules/redux/initRedux.js`
- page_b.js contains `docs/src/modules/redux/initRedux.js`

the JavaScript reference doesn't match, we end-up with two Redux store loading when transitioning from page a to page b, page b has no prior knowledge of the store of page a.

I imagine it be reproduced in dev mode, where we only bundle a few pages.

Reply:
Even if this were the case: What is the problem? These are pure functions (ignoring devtools). We can call them how often we want.

Reply:
And again: Every page could have its own module. We would still use the same store across pages since AppWrapper stays mounted.

Reply:
To illustrate: You're seeing the flamegraph of the commit after we navigated from / to /components/box. "ne" is the AppWrapper component that got updated. It didn't "render for the first time" i.e. mount. What mounted was the page (the big yellow bar):

![Screenshot from 2020-04-16 21-51-07](https://user-images.githubusercontent.com/12292047/79500367-967e5700-802c-11ea-9e39-f40c7c0bfd71.png)

The store wasn't re-created. That only happens when AppWrapper mounts which happens once.

Reply:
👌